packages/create/src/generators/app-haunted/index.js

```diff
@@ -0,0 +1,94 @@
+import path from 'path';
+import { executeViaOptions } from '../app/executeViaOptions.js';
+import { CommonRepoMixin } from '../common-repo/index.js';
+
+/* eslint-disable no-console */
+export const AppHauntedMixin = subclass =>
+  class extends CommonRepoMixin(subclass) {
+    async execute() {
+      await super.execute();
+
+      const { tagName, className } = this.templateData;
+
+      // user choosen tag name for app
+      const appMainOptions = {
+        ...this.options,
+        scaffoldType: 'wc-haunted',
+        tagName,
+        destinationPath: path.join(process.cwd(), tagName, 'packages', tagName),
+        noEnd: true,
+      };
+      delete appMainOptions.features;
```

Comment:
why?

Reply:
This file is mostly just a copy from https://github.com/open-wc/open-wc/blob/master/packages/create/src/generators/app-lit-element/index.js#L21